index.css

```diff
@@ -0,0 +1,315 @@
+@import url('https://fonts.googleapis.com/css2?family=Bree+Serif&family=Montserrat:ital,wght@0,400;0,500;0,900;1,600;1,900&family=Recursive:wght@300&family=Roboto+Slab&family=Source+Sans+Pro&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Bree+Serif&family=Montserrat:ital,wght@0,400;0,500;0,900;1,600;1,900&family=Recursive:wght@300&family=Roboto+Slab&family=Source+Sans+Pro&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Bree+Serif&family=Montserrat:ital,wght@0,400;0,500;0,900;1,600;1,900&family=Recursive:wght@300&family=Roboto+Slab&family=Source+Sans+Pro:ital@0;1&display=swap');
+
```

Comment:
Adding these links in the HTML head will give you a little bit of performance benefit.

Reply:
Okay i will look into it